.travis.yml

```diff
@@ -17,6 +17,8 @@ matrix:
   fast_finish: true
 
 install:
+  - pip install railroad-diagrams
```

Comment:
These two dependencies are supposed to be installed as "extras" ie `pip install pyparsing[diagrams]`. If you can work out how to tell Travis about extras, perhaps using a custom `install` key, this would be more robust

Reply:
For now, feels like more trouble than it is worth, since we manage both the travis.yml and the dependencies for diagrams.

Reply:
I think it should really be as simple as adding `install: pip install pyparsing[diagrams]`. The issue with this approach is that if these dependencies ever change, they would need to be updated in two different places.

Reply:
Or I guess since this stuff is done using tox (?), then using this  https://tox.readthedocs.io/en/latest/config.html#conf-extras

Reply:
Oh, it's already there. So I'll just remove the lines from .travis.yml.